### PR TITLE
feat: Add topo-imagery version information to STAC TDE-1265

### PR DESCRIPTION
**Pull request by l0b0**
### Motivation

As a data maintainer, geospatial data engineer or data consumer, I would like to reference the technical configuration used to create the TIFFs in our ODR datasets in order to be able to recreate the TIFFs or carry out troubleshooting.

### Modifications

Add `processing:software`, `processing:version`, and `processing:datetime` item properties.

### Verification

`pytest`

### CODE REVIEW

File: scripts/standardising.py [hunk 1]
```diff
@@ -50,11 +51,11 @@ def run_standardising(
 
     Returns:
         a list of FileTiff wrapper
+        :param gdal_version:
```
Comment (paulfouquet) [inline scripts/standardising.py:54]: Should be in the Args list?

File: README.md [hunk 2]
```diff
@@ -27,7 +27,7 @@ The scripts have been implemented to be run inside the Docker container only. Th
 - Build the `Docker` image:
 
 ```bash
-docker build . -t topo-imagery
+docker build --build-arg=GIT_HASH=dev --build-arg=GIT_VERSION=dev --tag=topo-imagery .
```
Comment (blacha) [inline README.md:30]: do we need to make the local build commands more complicated? what value does adding "dev"/"dev" add here?

File: scripts/standardising.py (hunk 1 above)
Comment (amfage) [reply scripts/standardising.py]: Oh good catch, how strange. IDEs gone rogue I think!

File: README.md (hunk 2 above)
Comment (amfage) [reply README.md:30]: There was some discussion about the best way to check that these exist when running in CI, open to suggestions! I think you are right we should change it as I can foresee they will end up commented out in the Dockerfile which would be a bad sign.

Comment (blacha) [reply README.md:30]: This is the first command in the README.md it is likely what people will run to first build this system, is making this command much more complicated to add "dev" "dev" seem excessive?

Comment (l0b0) [reply README.md:30]: The reason for making these mandatory was to ensure that we don't forget to add them, which would only be detected at runtime.

File: scripts/standardising.py (hunk 1 above)
Comment (l0b0) [reply scripts/standardising.py]: Apropos - what *is* the format we're using here called? Then I can probably configure IDEA to use that.

File: README.md (hunk 2 above)
Comment (l0b0) [reply README.md]: Changed to make these optional.

**Review by blacha — DISMISSED** [review]

**Review by blacha — APPROVED** [review]

**Pull request merged**